client-react/src/pages/app/change-app-plan/ChangeAppPlan.tsx

```diff
@@ -146,6 +112,128 @@ export const ChangeAppPlan: React.SFC<ChangeAppPlanProps> = props => {
 
   const subscriptionId = new ArmPlanDescriptor(currentServerFarm.id).subscription;
 
+  const getChangePlanHeaderCard = () => {
+    return (
+      <header>
+        <FeatureDescriptionCard name={t('changePlanName')} description={t('changePlanDescription')} Svg={AppServicePlanSvg} />
+      </header>
+    );
+  };
+
+  const getCurrentPlanDetails = () => {
+    return (
+      <>
+        <Stack style={sectionStyle}>
+          <h4 className={labelSectionStyle}>{t('changePlanCurrentPlanDetails')}</h4>
+        </Stack>
+
+        <ReactiveFormControl id="currentAppServicePlan" label={t('appServicePlan')}>
+          <div tabIndex={0} aria-label={t('appServicePlan') + getPlanName(currentServerFarm)}>
+            {getPlanName(currentServerFarm)}
+          </div>
+        </ReactiveFormControl>
+      </>
+    );
+  };
+
+  const getDestinationPlanDetailsHeader = () => {
+    return (
+      <Stack style={{ marginTop: '50px' }}>
+        <h4 className={labelSectionStyle}>{t('changePlanDestPlanDetails')}</h4>
+      </Stack>
+    );
+  };
+
+  const getPlanTypeRadioButton = (formProps: FormikProps<ChangeAppPlanFormValues>) => {
+    return (
+      <>
+        {Url.getFeatureValue(CommonConstants.FeatureFlags.enableFunctionsDynamicToPremium) === 'true' && (
+          <div className={planTypeStyle}>
+            <ReactiveFormControl id="planType" label={t('planType')}>
+              <RadioButtonNoFormik
+                id="planType"
+                ariaLabelledBy={t('planType')}
+                defaultSelectedKey={formProps.values.serverFarmInfo.newPlanInfo.tier}
+                options={[
+                  { key: ChangeAppPlanTierTypes.Dynamic, text: t('consumptionPlan') },
+                  { key: ChangeAppPlanTierTypes.ElasticPremium, text: t('functionPremiumPlan') },
+                ]}
+                onChange={(o, e) => e && handleTierChange(e.key as ChangeAppPlanTierTypes)}></RadioButtonNoFormik>
```

Comment:
Use self-closing tag.
```suggestion
                onChange={(o, e) => e && handleTierChange(e.key as ChangeAppPlanTierTypes)}> />
```